app/utils/file.js

```diff
@@ -46,6 +46,15 @@ export const isFileValid = (file, maxSizeInMb, fileTypes = []) => {
           case 'ffd8ffe2':
             type = 'image/jpeg';
             break;
+          case '25504446':
+            type = 'application/pdf';
+            break;
+          case 'd0cf11e0':
+            type = 'application/vnd.ms-powerpoint';
```

Comment:
Should we add support for LibreOffice formats too?

Reply:
@niranjan94 your views here?

Reply:
@niranjan94 I looked for MIME type for .odp files but couldn't find. I searched here also https://www.sitepoint.com/mime-types-complete-list/. Also given here on wikipedia https://en.wikipedia.org/wiki/List_of_file_signatures signature for pptx and odp files is same. Can we do something here?

Reply:
Sorry for previous comment. I found the MIME type.